plugins/foreman_ansible/3.x/index.md

```diff
@@ -0,0 +1,307 @@
+---
+layout: plugin
+title: Foreman Ansible 3.x Manual
+version: 3.x
+---
+
+# 1. Basics
+
+Foreman Ansible allows you to import hosts via Ansible, along with facts about these hosts and reports of their playbook runs. This plugin 3.x series should be compatible with any version of Foreman above 1.22, including 1.22. Foreman Ansible relies on [Foreman Remote Execution](plugins/foreman_remote_execution) in order to be able to run playbooks remotely.
+
+Every time you run a playbook or an Ansible module, Foreman will receive facts and a report for the host you have executed it on. If you ran Ansible from Foreman itself, the output of your Ansible run will be saved in our database.
+
+You can define a list of roles to enforce and run arbitrary playbooks on the hosts.
+![host with reports](static/images/plugins/foreman_ansible/registered_host.png)
+![rex results](static/images/plugins/foreman_ansible/rex_results.png)
+![rex playbook_output](static/images/plugins/foreman_ansible/rex_playbook_output.png)
+
+# 2. Installation
+
+## 2.1 Ansible callback
+
+<script type="text/javascript">
+function update_ansible_version(select) {
+  var version = select.value;
+  $(".ansible_version").hide();
+  if (version && version != 'none') {
+    $(".ansible_version_"+version).show();
+  } else {
+    $(".ansible_version_none").show();
+  }
+}
+</script>
+
+In order to make Ansible send us data from the hosts, we set up a callback on your host that runs Ansible.
+To provide specific installation instructions, please select your Ansible version:
+
+<select onChange="update_ansible_version(this);">
```

Comment:
IMHO we should hard require Ansible 2.5+ so there's only a single set of instructions. Reading these instructions it may even be 2.2+ since those instructions are the same.